sdk/cargo-build-sbf/src/main.rs

```diff
@@ -124,6 +125,61 @@ where
         .collect::<String>()
 }
 
+fn get_latest_sbf_tools_version() -> Result<String, String> {
+    let url = "https://github.com/solana-labs/sbf-tools/releases/latest";
+    let resp = reqwest::blocking::get(url).map_err(|err| err.to_string())?;
```

Comment:
Maybe help the user out with a bit more context when they hit this error due to bad wifi or something?
```suggestion
    let resp = reqwest::blocking::get(url).map_err(|err| format!("Failed to GET {}: {}", url, err.to_string()))?;
```